app/services/one_off_redirect_fix_service.rb

```diff
@@ -0,0 +1,27 @@
+class OneOffRedirectFixService
+  def self.fix_redirects!
+    editions = Edition.where(document_type: "redirect").joins(:document).where.not("documents.locale = 'en'")
+    editions.each do |edition|
+      update_redirects(edition)
+      Commands::V2::RepresentDownstream.new.call([edition.content_id], queue: DownstreamQueue::HIGH_QUEUE)
+    end
+  end
+
+  def self.update_redirects(edition)
+    return if edition.redirects.blank?
+
+    next_base_path = edition.redirects.first[:destination]
```

Comment:
We have to be careful assuming that the `redirects` array only has one element in it. I seem to remember that's almost always the case, but we should probably do a check here (and in the loop below) to exit if there is >1 redirect in the array, and log it in the job so that we can investigate manually later.

Reply:
done 👍